### DISCUSSION TITLE
2022-07-27 : R code port is functional

### DISCUSSION THREAD

**Discussion by 7yl4r** [USF-IMARS/chemtax#9] [Meeting Plans, Agendas, Notes]
* The R code port is (mostly) functional thanks to Sebastian.
* We need to secure approval before releasing publicly [see # 1](https://github.com/sebastiandig/rCHEMTAX/issues/1).
* Should start thinking about possible publication venues/topics
* Karen granted read access to R code. Simon will email github username to also get access.
* By next time we hope to have one (or more) of:
    * GUI ready to show
    * generalized R code able to take arbitrary input data
* We should publish an announcement paper
    * where?
    * will we do an example analysis?
        * does Enrique have ideas?
        * Karen will have marine science voyage data in 6mo
            * 26 sites
        * Simon works w/ 12yr x 4cruises per year surface samples hobart->antartica. Does cluster analysis on pigment ratios for each sample, then runs chemtax on clustered subsets.
    * paper should highlight new features eg
        * confidence intervals

**Comment by sebastiandig** [comment]
Thanks for these notes.  I will continue to work on making it more generalized. 

I do want to attempt to make it a package instead of just a repo with some functions in it. I think making it a package would be a better option when promoting it widely as they would be able to load the package and have all functions accessible as well as sample data, sample pigment ratios, standardized manuals, areas for testing, etc. 

I have started reading this webbook about creating [R packages](https://r-pkgs.org/). I hope this will give me the info to make this a very generalized packaged.
